src/main/java/com/github/invictum/panels/AbstractPanel.java

```diff
@@ -87,18 +87,13 @@ public Object evaluateJavascript(final String scriptToExecute, final Object... p
         return parentPage.evaluateJavascript(scriptToExecute, params);
     }
 
```

Comment:
It is better to make UnifiedDataProvider object protected and use it directly instead of creating new method.